src/components/partials/hostendpoints-upgrade.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import Admonition from "@theme/Admonition";
+import CodeBlock from "@theme/CodeBlock";
+
+function PriorToVersion(props) {
+	if (props.prodname === "Calico") {
+		return "v3.14";
+	}
+
+	return "v3.0";
+}
+
+function ApplyPolicyScript(props) {
+	if (props.prodname === "Calico") {
+		return (
+			<CodeBlock language="bash">
+				calicoctl apply -f - {"<"} allow-all-upgrade.yaml
+			</CodeBlock>
+		);
+	}
+
+	return (
+		<CodeBlock language="bash">
+			{props.orch === "OpenShift"
+				? "oc apply -f - < allow-all-upgrade.yaml"
+				: "kubectl apply -f - < allow-all-upgrade.yaml"}
+		</CodeBlock>
+	);
+}
+
+export default function HostEndpointsUpgrade(props) {
+	return (
+		<>
+			<h2 id="host-endpoints">Host Endpoints</h2>
+			<Admonition type="caution">
+				If your cluster has host endpoints with{" "}
+				<code>interfaceName: *</code> you must prepare your cluster
+				before upgrading. Failure to do so will result in an outage.
+			</Admonition>
+			<p>
+				In versions of {props.prodname} prior to{" "}
+				<PriorToVersion prodname={props.prodname} />, all-interfaces
+				host endpoints (host endpoints with{" "}
+				<code>interfaceName: *</code>) only supported pre-DNAT policy.
+				The default behavior of all-interfaces host endpoints, in the
+				absence of any policy, was to allow all traffic.
+			</p>
+			<p>
+				Beginning from <PriorToVersion prodname={props.prodname} />,
+				all-interfaces host endpoints support normal policy in addition
+				to pre-DNAT policy. The support for normal policy includes a
+				change in default behavior for all-interfaces host endpoints: in
+				the absence of policy the default behavior is to{" "}
+				<strong>drop traffic</strong>. This default behavior is
+				consistent with &quot;named&quot; host endpoints (which specify
+				a named interface such as &quot;eth0&quot;); named host
+				endpoints drop traffic in the absence of policy.
+			</p>
+			<p>
+				{/*
+          TODO [version]: Display correct version. From source:
```

Comment:
yeah, i took it out because it seemed unnecessary - we will likely have versioned docs, in which case, I don't know why adding a version here makes any difference. And, if we don't version our docs, it makes no sense. But, perhaps I'm missing something and I'll later realize we need it! :-)